main/src/ca/uwaterloo/flix/language/phase/Reducer.scala

```diff
@@ -200,7 +200,7 @@ object Reducer {
     * Returns all types contained in the given `Effect`.
     */
   private def typesOfEffect(e: Effect): Set[MonoType] = e.ops.toSet.map {
-    op: Op =>
+    (op: Op) =>
```

Comment:
Does it work without parens, but with braces?

Reply:
Can try

Reply:
It's already inside a call to `map` with braces.

Reply:
I can drop the type annotation, though.

Reply:
![Uploading CleanShot 2024-08-11 at 11.30.00@2x.png…]()

Reply:
Nevermind, the type annotation was required, but I moved it to separate function.